### ISSUE TITLE
Insufficient Color Contrast for Text in VSCode Interface

### ISSUE THREAD

**Issue by Bluefish0558** [microsoft/vscode#227492]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.88.0
- OS Version: 14.6.1 (23G93)

Steps to Reproduce: Certain text elements in the VSCode interface do not meet the required minimum color contrast ratio with their background, making it difficult for users with low vision to read the content.
**Issue Details:**
 ```
Foreground color: #858585
Background color: #FFFFFF
Contrast ratio: 3.69:1
Required contrast ratio: 4.50:1
```
 
**Examples of Affected Text:**
```
Commonly used
Text editor
Workbench
Window (hover state)
```
 
**Recommendation**: Ensure text and images of text provide sufficient contrast. The following contrast ratios are required:

* Text smaller than 18 pt (24 px) or smaller than 14 pt (19 px) if bold, must have a contrast ratio of 4.50:1 or more with adjacent colors.
* Text 18 pt (24 px) or larger, or 14 pt (19 px) or larger if bold, must have a contrast ratio of 3.00:1 or more with adjacent colors.
*  Disabled controls that do not accept user interaction are exempt from this requirement.
<img width="1247" alt="colorsettings" src="https://github.com/user-attachments/assets/b04fa105-a1f0-4cd2-b093-750d9296570e">

**Comment by connor4312** [comment]
What theme are you using?
